lib/KTable/index.vue

```diff
@@ -202,14 +203,6 @@
         type: String,
         required: true,
       },
-
-      /**
-       * Disables the default sorting when sortable is true. Facilitates integration with externally sorted data.
-       */
-      disableDefaultSorting: {
```

Comment:
Just a note for me we will need to mention `disableBuiltinSorting` -> `disableBuiltinSorting` renaming as a breaking change in the changelog.

Reply:
Another (not a strictly) breaking change would be the now mandatory requirement of `columnId` attribute in all the headers, if that may be good to mention in the changelog.

Reply:
Oh yes, definitely. Thank you